src/contexts/FastUnstake/index.tsx

```diff
@@ -180,7 +180,7 @@ export const FastUnstakeProvider = ({
 
       // ensure checked eras are in order highest first.
       const checked = metaRef.current.checked
-        .concat(currentEra)
+        .concat(new BigNumber(currentEra).toNumber())
         .sort((a: number, b: number) => b - a);
 
       if (!metaRef.current.checked.includes(currentEra)) {
```

Comment:
`currentEra` is still a string. Convert to `Number`:
```suggestion
      if (!metaRef.current.checked.includes(Number(currentEra))) {
```